services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/grid/internal/replica/view/query/filter/CellValueFilterElementTest.java

```diff
@@ -0,0 +1,61 @@
+package org.sagebionetworks.repo.manager.grid.internal.replica.view.query.filter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.sagebionetworks.repo.model.grid.query.CellValueFilter;
+import org.sagebionetworks.repo.model.grid.query.CellValueOperator;
+
+public class CellValueFilterElementTest {
+
+	/**
+	 * This is case that failed for PLFM-9309.
+	 */
+	@Test
+	public void testTransalteWithNullValue() {
+		CellValueFilter toClone = new CellValueFilter().setColumnName("c1").setOperator(CellValueOperator.IS_NOT_NULL);
```

Comment:
I think the operator IS_NOT_NULL might be confusing. Maybe it is better to have a dedicated filter instead? Or support c1 EQUALS null?